### PR TITLE
Add project goals to README

### PR DESCRIPTION
**Pull request by jackjennings**
I've added some language to describe what things are included, and what are not.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,21 @@
...
+* Processing submitted form data
+
+Microfrom doesn't explicitly handle:
+
+* Validations
+* Rendering inputs
+* Filtering parameters (`strong_params`)
+* Anything else
+
+Without these, the API for Microform can remain small (one controller method to remember), use plain old Ruby objects (without inheritance) that are easy to test and maintain. Even though Microform doesn't offer an opinion about the non-goals, it is possible to extend the simple Microform pattern to include any of these where it makes sense to in your application.
```
Comment (skwak) [inline README.md]: > "Even though Microform doesn't offer an opinion about the non-goals, it is possible to extend the simple Microform pattern to include any of these where it makes sense to in your application."

"any of these" in this case means Microform non-goals that someone might want to incorporate? Took me a second read to get that, if so. Might read better as "Even though Microsoft doesn't offer an opinion about the non-goals, it is possible to extend the simple Microform pattern to include them where it makes sense to in your application."

**Review by ghost — APPROVED** [review]
👍 I like this!

File: README.md (hunk 1 above)
Comment (jackjennings) [inline README.md]: `Microsoft` autocorrect ;)

I will update this!

Comment (skwak) [inline README.md]: Ahhahahaha! Oops

Comment (jackjennings) [inline README.md]: OK—I've updated this, and also changed "simple" to "basic", which is more appropriate?

Comment (skwak) [inline README.md]: I agree with your change — "basic" seems nicer as it focuses on the fundamentals of what a form "is" or "can do", while "simple" can also mean "easy-to-understand", which is not necessarily true.

Comment (jackjennings) [inline README.md]: Yep—I'm trying to remove "simple", "easy", etc. from documentation, but they're easy to fall back on when writing in a casual way… oops.

**Pull request merged**